core/src/test/java/dev/streamx/cli/ingestion/BaseIngestionCommandTest.java

```diff
@@ -0,0 +1,44 @@
+package dev.streamx.cli.ingestion;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static dev.streamx.clients.ingestion.StreamxClient.PUBLICATIONS_ENDPOINT_PATH_V1;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.github.tomakehurst.wiremock.junit5.WireMockExtension;
+import io.quarkus.test.junit.main.LaunchResult;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public abstract class BaseIngestionCommandTest {
```

Comment:
Introduced common base test class for repeated code such as setting up wiremock or endpoint paths